app/decorators/user_decorator.rb

```diff
@@ -88,9 +72,32 @@ def assigned_color
         text: "#c9d2dd"
       },
     ]
+    colors |= white_text
     colors[id % 10]
   end
 
+  def white_text
+    white = [
+      {
+        bg: "#093656",
+        text: "#ffffff"
+      },
+      {
+        bg: "#61122f",
+        text: "#ffffff"
+      },
+      {
+        bg: "#2e0338",
+        text: " #ffffff"
```

Comment:
this should probably have no leading space, I noticed it's present in master as well